webpack.config.js

```diff
@@ -25,7 +25,6 @@ module.exports = (env) => {
         },
         {
           test: /\.(scss|css)$/,
-          exclude: /node_modules/,
```

Comment:
This is for?

Reply:
By excluding the node_modules folder, webpack doesn't transform the files included in that folder. I imported the `ArtifaktElement.css` file into the project, which is in the node_modules folder. I removed that line so webpack could include the `ArtifaktElement.css.